data_pipelines_cli/cli_commands/deploy.py

```diff
@@ -156,12 +167,19 @@ def _sync_bucket(self) -> None:
     default=False,
     help="Whether to ingest DataHub metadata",
 )
+@click.option(
+    "--enable-ingest",
```

Comment:
From user perspective there might be some confusion. Should I set `--enable-ingest` flag when I would like to set `--datahub-ingest`?

Reply:
You know what, I'm thinking now... we don't want to control it with flag. Rather we would like to turn on/off based on ingestion.yml. Thanks to that we won't need to change CICD template.

Reply:
Hm interesting simplification idea. You mean the deploy command would behave as follows:
1. Lookup `enabled` value in ingestion.yml . 
2. If set to true, lookup file indicated by `engine` variable and perform factory actions

I assume the same algorithm should apply to BI files as well?

Reply:
exactly, I already wrote comment for Pawel chagnes